src/Makefile

```diff
@@ -10,7 +10,11 @@ CC ?= gcc
 SRCDIR = $(shell find . -type d -not -regex './obj.*' -printf '%P ')
 OBJDIR = $(patsubst %,obj/%,$(SRCDIR))
 
-LUASRC = $(shell find . -regex '[^\#]*\.lua' -printf '%P ')
+GENLUASRC := core/version.lua
+LUASRC = $(shell for f in $(GENLUASRC); do \
+	           if test ! -f $$f; then echo -n "$$f "; fi; \
```

Comment:
The intention was for the module to always exist; did I make an error?  The intention was for core/version.lua to be a "built source", like the lua modules that the dasl files compile into.

I didn't want to add version.lua to git because if the build process always overwrites it, then sadness would ensue.  WDYT?